### PR TITLE
[Bug] Fix point layer brushing

### PR DESCRIPTION
**Pull request by heshan0131**
kepler.gl manually edits deck.gl default layer shaders to enable brushing. This is sketchy practice overall since whenever deck.gl layer shader changes, brushing will break. 

After fixing the shader edit util for `scatterplot-brushing-layer`. I added test util to test render deck.gl layer using `@deck.gl/test-utils`. This way at least we fail the test if manually editing deck.gl shader errors out and we will catch it in the test.

### CODE REVIEW

**Comment by heshan0131** [comment]
#485

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,23 @@
 language: node_js
+addons:
```
Comment (macrigiuseppe) [inline .travis.yml]: are these used to test webgl in a server env?

Comment (heshan0131) [reply .travis.yml]: Yes, this is added because we have`gl` in devDependencies. And we run headless-gl test in travis

**Comment by heshan0131** [comment]
#484

**Pull request merged**